### PR TITLE
ENG-4917 merge release 7.2 branch into develop

### PR DESCRIPTION
**Pull request by ichalagashvili**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "app-builder",
-  "version": "7.2.0-SNAPSHOT",
+  "version": "7.2.1-SNAPSHOT",
```
Comment (entgigi) [inline package.json:3]: you could rewrite version as 7.3.0-SNAPSHOT in this PR to not open another PR ...

File: package-lock.json [hunk 2]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "app-builder",
-  "version": "7.2.0-SNAPSHOT",
+  "version": "7.2.1-SNAPSHOT",
```
Comment (entgigi) [inline package-lock.json:3]: you could rewrite version as 7.3.0-SNAPSHOT in this PR to not open another PR ...

File: Dockerfile [hunk 3]
```diff
@@ -5,7 +5,7 @@ LABEL name="Entando App Builder" \
       maintainer="dev@entando.com" \
       vendor="Entando Inc." \
       version="v${VERSION}" \
-      release="7.2.0" \
+      release="7.2.1" \
```
Comment (entgigi) [inline Dockerfile:8]: you could rewrite version as 7.3.0 in this PR to not open another PR ...

**Review by entgigi — APPROVED** [review]
I added some comments but basically LGTM

**Review by nicpuddu — APPROVED** [review]

**Pull request merged**